src/components/lead/CdrHeadingSubheadingBlock.vue

```diff
@@ -19,7 +19,10 @@ const style = useCssModule();
     >
       <slot />
     </CdrHeadingDisplay>
-    <CdrSubheadingSans :class="style[`${baseClass}__subheading`]">
+    <CdrSubheadingSans
```

Comment:
one question I have is around nesting the subhead span in the header - I think we may want to pass this by accessibility. I like the pattern but am also thinking that the heading may have a char limit for screen readers?

Reply:
I think it's okay, I used a similar pattern on the cedar doc site and ran it by @daemoncron.